lib/taskana-core-test/src/test/java/acceptance/task/update/UpdateTaskAccTest.java

```diff
@@ -569,19 +573,38 @@ private ObjectReferenceImpl createObjectReference(
 
   @Nested
   @TestInstance(Lifecycle.PER_CLASS)
-  class WithEnforceServiceLevelDisabled implements TaskanaConfigurationModifier {
+  class WithEnforceServiceLevelDisabledAndAdditionalUserInfoEnabled
+      implements TaskanaConfigurationModifier {
     @TaskanaInject TaskService taskService;
 
     @Override
     public TaskanaConfiguration.Builder modify(TaskanaConfiguration.Builder builder) {
-      return builder.enforceServiceLevel(false);
+      return builder.addAdditionalUserInfo(true).enforceServiceLevel(false);
+    }
+
+    @WithAccessId(user = "businessadmin")
+    @BeforeAll
+    void setup() throws Exception {
+      UserBuilder.newUser()
+          .id("user-1-2")
+          .firstName("Max")
+          .lastName("Mustermann")
+          .longName("Max Mustermann")
+          .buildAndStore(userService, "businessadmin");
+      UserBuilder.newUser()
+          .id("user-1-1")
+          .firstName("Ella")
+          .lastName("Mustermann")
+          .longName("Ella Mustermann")
+          .buildAndStore(userService, "businessadmin");
     }
 
     @WithAccessId(user = "user-1-2")
     @Test
     void should_AllowTimestampServiceLevelMismatch_When_ConfigurationAllowsIt() throws Exception {
       Task task =
           TaskBuilder.newTask()
+              .owner("user-1-2")
```

Comment:
why did you add this in this test?

Reply:
It was a copy-paste error from trying out before writing tests. I will remove it :)